libs/randomValueUtils.js

```diff
@@ -5,5 +5,8 @@ export const getRandomIntZeroToX = (x) => randomInt(0, x)();
 export const getRandomValueFromArray = (arr) =>
   arr instanceof Array && arr[getRandomIntZeroToX(arr.length)];
 
+export const getRandomValueDestructively = (arr) =>
```

Comment:
This is clever but you need to be really careful with stuff like this. If you don't copy the original array you will mutate it and that could cause bugs